src/main/java/com/shibumi/MaximumPathSum.java

```diff
@@ -27,7 +27,7 @@ public int calculateMaxSum() {
         return memo[matrix.length-1][matrix[0].length-1];
     }
 
-    private void print(int[][] array) {
+    public void print(int[][] array) {
```

Comment:
It feels like this method should either:
  a. be static
  b. use this.matrix, rather than taking the matrix to print as an argument.

Reply:
I created the method as part of my first implementation, even before the algorithm worked, I passed  a parameter so it can be called from my test if I wanted to print my sample input, given the use I think making it static makes the most sense, ideally it would be in it's own  utility class.